### PR TITLE
DEVP-3643 rename header

### PR DESCRIPTION
**Pull request by MickStanciu**
## Summary

rename the header

## Checklist

- [x] Is this PR a reasonable size?

- [ ] List deployment sequence with all relevant PRs.
  - ...

- [x] For breaking changes: I have confirmed that other repos are not affected, or I have planned a deployment sequence to address any impact.

---

**Code Review Guidelines** for Reviewers

- Try to review in a timely manner. Opinions/nitpicks should not be blockers. Pair on a call for non-trivial feedback.
- Overall design and approach should follow established patterns. Don't try to make the PR perfect.
- Try to identify edge cases, race conditions, over-engineering, lack of test coverage and complexity.
- If you don't feel qualified to review the code, pass it on to someone who is.

### CODE REVIEW

File: s12/protobuf/workato/annotations.pb.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.36.4
+// 	protoc-gen-go v1.34.1
```
Comment (HarryTranAU) [inline s12/protobuf/workato/annotations.pb.go:3]: version going down?

Comment (MickStanciu) [reply s12/protobuf/workato/annotations.pb.go:3]: ??? not sure, that is autogenerated. It is a file for tests

**Review by HarryTranAU — APPROVED** [review]
